lessons/pydata/webscraping/index.ipynb

```diff
@@ -0,0 +1,6844 @@
+{
```

Comment:
<p>To se mi líbí, je to k tématu, protože tímhle se už (bez detailů) zabýváme už v EDA2 - ani tam ČR není na prvním místě :-) Možná by stálo za to na nějak "vesele" odkázat. Zajímavé, že je ta tabulka tak nějak divně poskládaná (různé sloupce z různých let), a že anglická verze má jiná čísla.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>